SETUP/tests/UnicodeTest.php

```diff
@@ -13,6 +19,48 @@ public function testSubstrReplace()
         $this->assertEquals("ЖЖbbc", $new_string);
     }
 
+    public function testUtf8CombinedChr()
+    {
+        $codepoint = 'U+004E';  # N
+        $char = utf8_combined_chr($codepoint);
+        $this->assertEquals("N", $char);
+    }
+
+    public function testUtf8CombinedChrCombined()
+    {
+        $codepoint = 'U+004E>U+0305';  # N̅
+        $char = utf8_combined_chr($codepoint);
+        $this->assertEquals("N̅", $char);
```

Comment:
We should really write this as
```
$this->assertEquals("\u{4e}\u{305}", $char);
```
It doesn't make any difference in this case but if we did the test for 'U+004E>U+0307'
and wrote the assertion for
```
 "\u{4e}\u{307}"
```
then it would fail because the function would normalise it to 
```
"\u{1e44}"
```
although both look like "Ṅ"

Reply:
Good call - fixed.